sessionctx/variable/sysvar.go

```diff
@@ -628,6 +628,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeGlobal, "innodb_online_alter_log_max_size", "134217728"},
 	{ScopeSession, WarningCount, "0"},
 	{ScopeSession, ErrorCount, "0"},
+	{ScopeGlobal | ScopeSession, "information_schema_stats_expiry", "86400"},
```

Comment:
we should also add tests for setting and getting this session/global variable.

Reply:
Done (as expression/integration tests.)